### PR TITLE
Add validation for 'Verify win' stage of investment projects

### PR DESCRIPTION
**Pull request by reupen**
This also changes how the validation works so it's simple and more maintainable.

### CODE REVIEW

File: datahub/core/utils.py [hunk 1]
```diff
@@ -11,6 +11,11 @@
 logger = getLogger(__name__)
 
 
+def is_falsey(val):
+    """Returns True if bool(val) is False"""
+    return not val
```
Comment (canni) [inline datahub/core/utils.py]: RLY, why not a plain lambda? ;]

Comment (reupen) [inline datahub/core/utils.py]: DRY? :wink:

Comment (canni) [inline datahub/core/utils.py]: Then maybe run everywhere `s/if not (.+):/if is_falsey($1)/` ;]

Comment (reupen) [inline datahub/core/utils.py]: operator.not_() did the same thing, so it's gone now.

File: datahub/investment/validate.py [hunk 2]
```diff
@@ -52,11 +52,11 @@
     'non_fdi_type':
         CondValRule('investment_type', InvestmentType.non_fdi.value.id, Phase.prospect.value),
     'total_investment':
-        CondValRule('client_cannot_provide_total_investment', is_falsey, Phase.assign_pm.value),
+        CondValRule('client_cannot_provide_total_investment', not_, Phase.assign_pm.value),
```
Comment (canni) [inline datahub/investment/validate.py]: :+1:

File: datahub/investment/test/test_validate.py [hunk 3]
```diff
@@ -191,8 +204,96 @@ def test_validate_team_instance_success():
...
+        'r_and_d_budget': 'This field is required.',
+        'non_fdi_r_and_d_budget': 'This field is required.',
+        'new_tech_to_uk': 'This field is required.',
+        'export_revenue': 'This field is required.',
+        'address_line_1': 'This field is required.',
+        'address_line_2': 'This field is required.',
+        'address_line_postcode': 'This field is required.',
+        'average_salary': 'This field is required.',
+        'client_cannot_provide_foreign_investment': 'This field is required.',
+        'foreign_equity_investment': 'This field is required.',
```
Comment (elcct) [inline datahub/investment/test/test_validate.py]: Do we need to repeat `This field is required.` like that?

Comment (reupen) [inline datahub/investment/test/test_validate.py]: Not sure what you mean – what is your suggestion?

Comment (elcct) [inline datahub/investment/test/test_validate.py]: That way it doesn't immediately tells that the responses are identical. I would either use constant `REQUIRED_MESSAGE` or create a variable `this_field_is_required` and assert against it. 
For example:
```
...
'address_line_postcode': REQUIRED_MESSAGE,
'average_salary': REQUIRED_MESSAGE,
...
```

Comment (reupen) [inline datahub/investment/test/test_validate.py]: It doesn't make much difference to me, but I kinda like seeing the output as is in tests.

Comment (elcct) [inline datahub/investment/test/test_validate.py]: To me it hints that maybe one of the responses is different than the others in a subtle way (for example missing dot or a typo), which you can't tell without looking more closely.

**Pull request merged**